src/app/pages/auth/services/keycloak.auth.service.ts

```diff
@@ -238,6 +244,7 @@ export class KeycloakAuthService extends AuthService {
     const responseMode = 'query'
     const responseType = 'code'
     const scope = 'openid'
+    this.language = this.localization.getLanguage()
```

Comment:
Isn't this an asynchronous call? If it is, maybe adding the return and then would be better

Reply:
No, it is not async call.
```localization.service.ts```
```js
getLanguage(): LanguageSetting {
    return this.language
  }
```